frontend/src/Editor/Components/Table/Table.jsx

```diff
@@ -455,6 +455,21 @@ export function Table({
     },
     [serverSidePagination, clientSidePagination, setPaginationInternalPageIndex]
   );
+  registerAction(
+    'selectRow',
+    async function (key, value) {
```

Comment:
Please see the video. Try these steps to avoid the issue.

- Use table's data from `tableData` instead of currentState to filter to the desired row. (Remember that we're trying to avoid `currentState` as much as possible)
- Pass the stringified version of that dependency also

https://user-images.githubusercontent.com/20333972/201629853-03a933c9-bcf0-45ff-8e8c-dddcd1623d85.mov